packages/qdrant-loader/src/qdrant_loader/core/state/state_change_detector.py

```
@@ -112,10 +113,85 @@ async def detect_changes(
 
         return changes
 
+    async def classify_batch(
+        self,
+        documents: list[Document],
+        filtered_config: SourcesConfig,
```

Ruff (0.15.14)

[warning] Unused method argument: `filtered_config`

(ARG002)